examples/bri-3/src/bri/merkleTree/agents/merkleTreeStorage.agent.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@nestjs/common';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+import { PrismaService } from '../../../../prisma/prisma.service';
+import MerkleTree from 'merkletreejs';
+import { BpiMerkleTree } from '../models/bpiMerkleTree';
+
+@Injectable()
+export class MerkleTreeStorageAgent extends PrismaService {
+  constructor(@InjectMapper() private mapper: Mapper) {
+    super();
+  }
+
+  async getMerkleTreeById(id: string): Promise<BpiMerkleTree> {
+    const merkleTreeModel = await this.bpiMerkleTree.findUnique({
+      where: { id },
+    });
+
+    if (!merkleTreeModel) {
+      return null;
```

Comment:
we can log here. Additionally, should it be undefined or null?

Reply:
following the repo standard here, I believe this was introduced by @ognjenkurtic  for reasons of not throwing exceptions at storage agent level. The same impl is used widely in other storage agents within the repo